BunnymodXT/modules/HwDLL.cpp

```diff
@@ -3758,16 +3833,48 @@ struct HwDLL::Cmd_BXT_FreeCam
 	}
 };
 
+void HwDLL::PrintEntities(std::ostringstream &out, int e, const edict_t* ent)
```

Comment:
```suggestion
void HwDLL::PrintEntity(std::ostringstream &out, int index)
```
It can get edict itself, especially since it already gets classname and whatnot.